### PR TITLE
Fixes: Issue #37 and Issue #65

### PR DESCRIPTION
**Pull request by thomasquintana**
test-one-file.lua broken

### CODE REVIEW

File: test-one-file.lua [hunk 1]
```diff
@@ -5,68 +5,218 @@
...
+local fwav = sndfile.SndFile(opt.wav)
 local fwavinfo = fwav:info()
 print(string.format('| number of frames: %d (%6.2fs) [samplerate: %d channels: %d]',
                     fwavinfo.frames, fwavinfo.frames/fwavinfo.samplerate, fwavinfo.samplerate, fwavinfo.channels))
 fwav:close()
 
 -- note that the current setup expect the whole file as a ByteTensor
-local fwav = torch.DiskFile(testopt.wav):binary():quiet()
-local wav = torch.ByteTensor(fwav:readByte(2^30))
+local wav = audio.load(opt.wav)
```
Comment (vineelpratap) [inline test-one-file.lua]: `sndfile` package should be able to do this. Adding `audio` dependency seems unnecessary.

**Comment by vineelpratap** [comment]
Hi, Looks good to me. Thanks ! Just added one comment about adding additional dependencies. In general, we want to reduce them as much as possible. I'll also try to test it on my local machine soon.

File: test-one-file.lua (hunk 1 above)
Comment (thomasquintana) [reply test-one-file.lua]: Cool, I will fix this over the weekend and create a separate pull request for a docker container. -- Cheers!

Comment (vineelpratap) [reply test-one-file.lua]: ```local wav = fwav:readFloat(fwav:info().frames)``` works here.

**Comment by vineelpratap** [comment]
Hi, I just made a comment. Other than that the code looks fine and good to go !

File: test-one-file.lua [hunk 2]
```diff
@@ -5,68 +5,218 @@
...
+          table.insert(str, letter)
+       end
+    )
+    return table.concat(str)
+ end
 
 -- basic info
-local fwav = sndfile.SndFile(testopt.wav)
+local fwav = sndfile.SndFile(opt.wav)
 local fwavinfo = fwav:info()
```
Comment (vineelpratap) [inline test-one-file.lua]: can you output error here if `fwavinfo.samplerate != opt.samplerate`

File: test-one-file.lua [hunk 3]
```diff
@@ -5,68 +5,218 @@
 -- LICENSE file in the root directory of this source tree. An additional grant
 -- of patent rights can be found in the PATENTS file in the same directory.
```
Comment (vineelpratap) [inline test-one-file.lua]: You can add a line or two here on how to use it.
1. Follow the steps in - https://github.com/facebookresearch/wav2letter#running-the-decoder-inference
2. Example usage on librispeech pre-trained model - ~luajit ~/test-one-file.lua <PATH_TO_MODEL> <PATH_TO_WAV_FILE> -letters ~/librispeech-proc/letters-rep.lst  -words ~/dict.lst -lm ~/3-gram.pruned.3e-7.arpa -lmweight 3.1639 -beamsize 25000 -beamscore 40 -nthread 10 -smearing max

**Comment by thomasquintana** [comment]
Hi, I apologize. I have been a bit wrapped up with work but I will get these done this week and create a new pull request. Cheers!

**Comment by vineelpratap** [comment]
Thanks @thomasquintana !

**Pull request merged**